# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/statusfilter/nsBrowserStatusFilter.cpp
# Commit: 36b0646e8649
# Full Hash: 36b0646e86498662e770756ad54b1fd648a82184
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2020-04-07 21:44:02
# Regressor Bug: 1620594
# File Overlap Count: 1
# Description:
#   Bug 1620594 - Part 7: Remove TabGroup and SystemGroup. r=nika,bas
#   
#   TabGroup never really made any difference in which thread something go
#   dispatched to. This was the intended use, but development of TabGroups
#   with abstract main threads never made it that far. The good thing is
# ==============================================================================

diff -r a5ad3b179c92 -r 36b0646e8649 toolkit/components/statusfilter/nsBrowserStatusFilter.cpp
--- a/toolkit/components/statusfilter/nsBrowserStatusFilter.cpp	Tue Apr 07 15:17:14 2020 +0000
+++ b/toolkit/components/statusfilter/nsBrowserStatusFilter.cpp	Tue Apr 07 15:17:47 2020 +0000
@@ -4,7 +4,6 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "nsBrowserStatusFilter.h"
-#include "mozilla/SystemGroup.h"
 #include "nsITimer.h"
 #include "nsString.h"
 #include "nsThreadUtils.h"